src/components/VButton.vue

```diff
@@ -95,7 +101,7 @@ const VButton = defineComponent({
     as: {
       type: String as PropType<ButtonForm>,
       default: 'button',
-      validate: (val: ButtonForm) => buttonForms.includes(val),
+      validate: (val: unknown) => buttonForms.includes(val as ButtonForm),
```

Comment:
TIL!

Reply:
I only changed this because I was trying to get the `ExtractPropTypes` to work correctly with props that have default values. I'll set it back, there's no functional or operational difference either way.

Reply:
(Though the `unknown` parameter type is _technically_ more correct).

Reply:
It just doesn't matter because we don't call `validate` directly so the type checking in this function is basically a "whatever" kind of thing.